FirebaseMLModelDownloader/Sources/ModelDownloader.swift

```diff
@@ -332,17 +335,20 @@ extension ModelDownloader {
 
 /// Possible error messages while using model downloader.
 extension ModelDownloader {
-  private static let defaultAppNotConfiguredErrorDescription =
-    "Default Firebase app not configured."
-  private static let parseModelNameErrorDescription = { (path: String) in
-    "Unable to parse model file name at \(path)."
-  }
+  /// Error descriptions.
+  private enum ErrorDescription {
+    static let defaultAppNotConfiguredError =
+      "Default Firebase app not configured."
+    static let parseModelNameError = { (path: String) in
+      "Unable to parse model file name at \(path)."
+    }
 
-  private static let retrieveLocalModelInfoErrorDescription =
-    "Failed to get stored model info for model file."
-  private static let outdatedModelPathErrorDescription = "Outdated model paths in local storage."
-  private static let deletedLocalModelInfoErrorDescription =
-    "Model unavailable due to deleted local model info."
-  private static let backgroundModelDownloadErrorDescription: StaticString =
-    "Failed to update model in background."
+    static let retrieveLocalModelInfoError =
+      "Failed to get stored model info for model file."
+    static let outdatedModelPathError = "Outdated model paths in local storage."
+    static let deletedLocalModelInfoError =
+      "Model unavailable due to deleted local model info."
+    static let backgroundModelDownloadError: StaticString =
+      "Failed to update model in background."
+  }
```

Comment:
Nit: the `Error` appended at the end isn't really necessary anymore since it's prefixed with `ErrorDescription.`

Reply:
Gotcha, done. Some of them maybe sound a little vague now, e.g. "ModelInfoRetriever.ErrorDescription.serverResponse" (which is for server responses with HTTP error codes). Is that ok or do you have any suggestions on how to rename something like that? Thanks!